policyengine_us/parameters/gov/irs/credits/ctc/adult_dependent_amount.yaml

```diff
@@ -1,11 +1,14 @@
 description: Maximum value of the CTC for adult dependents.
 values:
-  2017-01-01: 500
+  0000-01-01: 0
+  2018-01-01: 500
+  2026-01-01: 0
 metadata:
   unit: currency-USD
   period: year
   name: ctc_adult_dependent
   label: Child tax credit for adult dependents
   reference:
+    # TCJA only.
```

Comment:
Should we add a "comment" field in Core?

Reply:
Yes, and/or a start/end period? This would duplicate the functionality of providing references for a specific period, but reduce code duplication when there are breakdowns. Alternatively, flipping the order of dates and breakdowns...